.github/workflows/generate-sdk.yml

```diff
@@ -408,6 +442,23 @@ jobs:
       - uses: ./.github/workflows/composite/test-sdk
         with:
           test-suite: TestBillingApiTestSuite
+  test-networkstorageapi:
+    name: Test Billing API
```

Comment:
Change name to `Test Network Storage API`

Reply:
🤦